### ISSUE TITLE
Add cinebench

### ISSUE THREAD

**Issue by musm** [microsoft/winget-pkgs#17226]
Add 
 https://www.maxon.net/en/downloads/cinebench-r23-downloads/

# Package Requested

- [x] I would like someone else to build the manifest.

Manifest:

Fill out as much of the manifest metadata you can:

```YAML
PackageIdentifier:   # publisher.package format (example: "Microsoft.WindowsTerminal")
PackageVersion:      # version number format x.y.z.a (example: "1.6.10571.0")
PackageLocale:       # meta-data locale (example: "en-US")
Publisher:           # publisher name (example: "Microsoft Corporation")
PackageName:         # package name (example: "Windows Terminal")
License:             # package license (example: "MIT")
ShortDescription:    # package description (example: "The new Windows Terminal")
Installers: 
 - Architecture:     # installer architecture (example: "x64")
   InstallerType:    # installer type (example: "msix")
   InstallerUrl:     # installer URL (should be https://)
   InstallerSha256:  # SHA256 hash calculated from installer
ManifestType: singleton
ManifestVersion: 1.0.0
```

**Comment by musm** [comment]
This is blocked by the fact that the installer in a zip format.

**Comment by Trenly** [comment]
Cinebench is available through the msstore source - `winget install 9pgzkjc81q7j`

**Comment by CanePlayz** [comment]
If everything works out fine, it will be added with #99328

**Comment by CanePlayz** [comment]
@denelon This issue can be closed.

**Comment by ItzLevvie** [comment]
Closes with reason: Resolved by PR 99328;

**Comment by Trenly** [comment]
Resolved in #99328

**Comment by CanePlayz** [comment]
> Resolved in #99328

Can I just type that out and the bot will close that issue or is that a special power of yours? :P
